support-frontend/assets/helpers/user/user.ts

```diff
@@ -1,19 +1,15 @@
 // ----- Imports ----- //
-import type { ThunkDispatch } from 'redux-thunk';
+import { getGlobal } from 'helpers/globalsAndSwitches/globals';
 import * as cookie from 'helpers/storage/cookie';
-import { get as getCookie } from 'helpers/storage/cookie';
-import { getSession } from 'helpers/storage/storage';
 import type { Option } from 'helpers/types/option';
 import { getSignoutUrl } from 'helpers/urls/externalLinks';
-import { routes } from 'helpers/urls/routes';
-import { defaultUserActionFunctions } from 'helpers/user/defaultUserActionFunctions';
-import type { Action, UserSetStateActions } from 'helpers/user/userActions';
 
 export type User = {
 	firstName: Option<string>;
 	lastName: Option<string>;
 	email: Option<string>;
 	isSignedIn: boolean;
+	address4?: string;
```

Comment:
Not sure what this is? Some sort of 4th address row field (for testing)? If empty, we load with stateCode? Is there a better name or perhaps need to go changing Scala code perhaps?

Reply:
`address4` should be the fourth line of the address if we have it, so state/province/county. This is information that is added to `window.guardian` from the back end, we don't update it or set it in client-side code. I'm not sure where the names of the properties come from but it's likely MDAPI or Identity, so not trivial to change it.

Reply:
Ok